.travis.yml

```diff
@@ -0,0 +1,51 @@
+os:
```

Comment:
can we use buildkite for macos? this introduces the third pipeline (github workflows, buildkite and travis ci)

Reply:
It's tricky with Buildkite, because our approach is based around Docker containers, but we cannot run macOS in Docker.  Travis CI has some benefits here because it's free and provides macOS VMs out of the box.